### ISSUE TITLE
Alignment of World Times board

### ISSUE THREAD

**Issue by redburn** [phaselden/FlipIt#7]
The World Times board is not quite centered on the screen. Here's a screenshot with grid lines:

![2021-01-24-14_21_27-Greenshot](https://user-images.githubusercontent.com/3878542/105631813-9e3b1980-5e50-11eb-8ccb-322b2b5e6ac8.png)

(I increased the contrast in the screenshot to make it more noticeable.)

**Comment by phaselden** [comment]
Yes thanks. That's on the todo list. 

I also notice the last 2 entries have MA at the end. I assume that's the MMM value for Monday in your locale. I had assumed that would always output 3 chars.

**Comment by redburn** [comment]
Great! And yes, in Dutch (and German) the day abbreviations are always 2 characters long.

Two other minor things related to World Times, possible also already on your to-do list:
* when the 12-hour format is set, AM/PM is not displayed here.
* when adding a city to the list, e.g. Tokyo, I have to select Tokyo from the dropdown before I've finished typing Tokyo because the dropdown disappears as soon as Tokyo is fully entered (and I can't add it to the list that way).

**Comment by phaselden** [comment]
After a little investigation, it seems that in the "nl_NL" culture, the AM or PM is written as "a.m." or "p.m.". So each is 4 chars long whereas the code is expecting 2 chars. which would explain why it's not appearing. I guess I have 2 choices: Either force it to be "PM" or make room for it on the board. At the moment I am tempted to do the former, mainly because it would be consistent with the Current Time screen.

**Comment by redburn** [comment]
I'd also opt for consistency. Anyone who reads time in the 12-hour format will understand AM/PM.

**Comment by redburn** [comment]
The alignment is more consistent now in 1.2, thanks!

Maybe it's just me, but when the MMM value isn't displayed on the right, the board does still look a bit misaligned because the "flaps" in the background are quite a dark gray and there's not much padding around the board as a whole:

![2021-01-27 11_50_54-Greenshot](https://user-images.githubusercontent.com/3878542/105982247-535f1300-6097-11eb-88db-c7a79f6ccfc2.png)

It's less noticeable when the background is lighter, but it's still quite tight against the borders of the screen:

![2021-01-27-11_50_54-Greenshot](https://user-images.githubusercontent.com/3878542/105982289-5f4ad500-6097-11eb-82e7-0716ef1a2679.png)

I'm nitpicking obviously, but maybe it can be sized down just a bit (perhaps via a setting) and/or the background made a little lighter?

> when adding a city to the list, e.g. Tokyo, I have to select Tokyo from the dropdown before I've finished typing Tokyo because the dropdown disappears as soon as Tokyo is fully entered (and I can't add it to the list that way).

This works great now!

**Comment by phaselden** [comment]
Yeah I noticed the padding had disappeared just before releasing 1.2. I will look in to that. 

I did make the background the darker colour used a week or so ago, I might switch it back to the lighter colour. 

Re the MMM, I have been wondering whether I should have an option to display a + or - in the DST box instead of always having the 4 extra boxes. 
Related to this... another ticket asks for the ability to turn the DST asterisk off, but even with it on, the +/- could work if displayed in the bottom half of the box.

**Comment by redburn** [comment]
Not sure if the + or - would be intuitive enough. I don't think I'd understand it immediately (could be my ignorance, of course).

You could also opt to always show the MMM, but it might make it a bit more crowded.

**Comment by jsonjlee** [comment]
@phaselden Is there any way I can download this on the MacOS?

I'm using MacOS Big Sur.

Would love to get this as my screensaver!

**Comment by SmilerRyan** [comment]
> @phaselden Is there any way I can download this on the MacOS?
> 
> I'm using MacOS Big Sur.
> 
> Would love to get this as my screensaver!

@jsonjlee For the moment, I'd believe the best option is to install **Wine** on there, and run the screensaver file (maybe from the desktop) every time you want it open.

**Comment by jsonjlee** [comment]
Could you actually make another file for MacOS?

I can't use wine because their version is not available for my operating system version.

**Comment by phaselden** [comment]
Hi @jsonjlee. 

This does not work with MacOS. Getting it to work might be a bit of a challenge as (I assume) the way they implement screen savers is quite different from how it is done on Windows. Also, the .NET Framework and WinForms we are currently using does not work on MacOS.

Have you looked at [Fliqlo](https://fliqlo.com/screensaver/)? That project works on Mac and was the original inspiration for FlipIt.
